vald/config/testdata/config.toml

```diff
@@ -9,6 +9,7 @@ min_sleep_before_retry = "1ns"
 name = "evm-1"
 rpc_addr = "https://localhost:7475"
 start-with-bridge = true
+require_confirmation_height = true
```

Comment:
I would prefer a more generic option such as `finality_method` with two options available rn `auto`, `confirmations`. This will allow adding other options more easily in the future

Reply:
how we know if validator specifies an incorrect option though? we can only check invalid inputs